config

```diff
@@ -134,16 +138,25 @@ EOF
 
 if ask_no "Do you want to install needed R packages now?"
 then
-	salloc $(arg -p $DEBUGQ) --ntasks=1 $PP/exec.install skipssl rdep
+	if ARE_U_ON_RYSY
```

Comment:
There should not be any cluster specific code outside of `cluster/...` Remember that `arg` function works in that way, that if `$DEBUG_QOS` is empty then `$(arg --qos $DEBUG_QOS)` will be empty (look: https://github.com/CFD-GO/TCLB_cluster/blob/89d1366fb06cf12316845bf79e07c86e00b083ab/lib.bash#L188)

Reply:
done